website/content/docs/ui/custom-login.mdx

```diff
@@ -0,0 +1,538 @@
+      --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+      --header "X-Vault-Namespace: ${VAULT_NAMESPACE}" \
+      --data @data.json                         \
+      ${VAULT_ADDR}/v1/sys/auth/gui/ldap | jq
+  ```
+
+  </CodeBlockConfig>
+
+  `/sys/auth/{plugin_mount_path}` does not return data on success.
+
+</Tab>
+</Tabs>
+
+### Option 2: Update an existing plugin
+
+<Tabs>
+<Tab heading="CLI" group="cli">
+
+Use `vault auth tune` to update the `-listing-visibility` flag to `unauth`:
+    
+```shell-session
+$ vault auth tune              \
+  -namespace <plugin_namespace>  \
+  -description <visibility_note> \
+  -listing-visibility "unauth"   \
+  <mount_path>
+```
+    
+For example:
+    
+<CodeBlockConfig hideClipboard="true">
+
+```shell-session
+$ vault auth tune                        \
+  -namespace admin                       \
+  -description "Default GUI auth method" \
+  -listing-visibility "unauth"           \
+  gui/ldap
+
+Success! Tuned the auth method at: gui/ldap/
+```
+
+</CodeBlockConfig>
+
+</Tab>
+<Tab heading="GUI" group="gui">
+
+@include 'gui-instructions/update-authn-plugin.mdx'
+
+- Click **Configure >**.
+- Enable **Use as preferred UI login method** toggle.
+- Optional: Update **Description** to note the method as a default or preferred
+  authentication method for the GUI.
+- Click **Update options**.
+
+</Tab>
+<Tab heading="API" group="api">
+
+1. Create a JSON file with the updated configuration information for your
+   authentication plugin and set the `listing_visibility` parameter to `unauth`.
+   
+   For example:
+
+    ```json
+    {
+      "description": "Default GUI auth method",
+      "listing_visibility": "unauth"
+    }
+    ```
+
+1. Make a `POST` call to
+   [`/sys/auth/{plugin_mount_path}/tune`](/vault/api-docs/system/auth#tune-auth-method)
+   with the JSON data:
+
+    ```shell-session
+    $ curl                                             \
+      --request POST                                   \
+      --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+      --header "X-Vault-Namespace: ${VAULT_NAMESPACE}" \
+      --data @data.json                                \
+      ${VAULT_ADDR}/v1/sys/auth/<plugin_mount_path>/tune
+    ```
+
+    For example:
+
+    <CodeBlockConfig hideClipboard>
+
+    ```shell-session
+    $ curl                                               \
+        --request POST                                   \
+        --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+        --header "X-Vault-Namespace: ${VAULT_NAMESPACE}" \
+        --data @data.json                         \
+        ${VAULT_ADDR}/v1/sys/auth/gui/userpass/tune | jq
+    ```
+
+    </CodeBlockConfig>
+
+    `/sys/auth/{plugin_mount_path}/tune` does not return data on success.
+
+</Tab>
+</Tabs>
+
+## Step 3: Set your default and preferred authentication methods
+
+The following authentication method types are supported:
+
+@include 'ui/supported-login-methods-api-docs.mdx'
+
+<Tip>
```

Comment:
I wasn't really sure where to put this `<Tip>`
<img width="724" alt="Screenshot 2025-06-02 at 2 04 23 PM" src="https://github.com/user-attachments/assets/30e7a9f0-01ab-45ab-aa42-b196abfedd90" />